app/helpers/codidact_helper.rb

```diff
@@ -0,0 +1,11 @@
+# Provides helper methods to determine whether this site is part of the codidact network
+module CodidactHelper
+  # Whether this server is part of the codidact network.
+  def codidact?
+    Rails.cache.fetch 'is_codidact' do
```

Comment:
I wonder whether the Redis cache is going to be faster than checking the string twice. It might be premature optimization.